dotCMS/src/main/java/com/dotcms/util/transform/DBTransformer.java

```diff
@@ -1,6 +1,7 @@
 package com.dotcms.util.transform;
 
 import java.util.List;
+import java.util.Optional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)